nix/desktop/base-image/default.nix

```diff
@@ -0,0 +1,56 @@
+{ stdenv, fetchurl, unzip }:
+
+let
+  defaultPackageSource = {
+    version = "20190923";
+    hostSystem = "x86_64-linux";
+  };
+  packageSources = {
+    "linux" = defaultPackageSource // {
+      sha256 = "1l2rmqc2mxlz4qp5pnl1763pzqh6y3aawxrd9336q5w35xgrgmcm";
+    };
+    "macos" = defaultPackageSource // {
+      sha256 = "1j73l7xry0xw922zxhbsai1391a4i15rni1pfszr3cis8v95n21v";
+      hostSystem = "x86_64-darwin";
+    };
+    "windows" = defaultPackageSource // {
+      sha256 = "0sb1nqwy2ap7lr1vrk497fqrjhv7c470pm0kmrvwn4nas4gm40g5";
+    };
+  };
+  packageFactory = target-os:
+    let packageSource = packageSources."${target-os}";
+    in stdenv.mkDerivation rec {
+      inherit (packageSource) version;
+      pname = "status-im-${target-os}-desktop-files";
+
+      src = assert stdenv.lib.asserts.assertMsg (stdenv.hostPlatform.system == packageSource.hostSystem) "${packageSource.pname} is not supported on ${stdenv.hostPlatform.system}";
```

Comment:
Wait, isn't `packageSource.pname` wrong? I seems to be set right before `src` as just `pname` or am I missing something?

Reply:
Awesome catch! When merging my script from develop into this branch I missed this bit, thanks!